RecoJets/Configuration/python/RecoPFJets_cff.py

```diff
@@ -61,6 +61,9 @@
     )
 
 
+fixedGridRhoFastjetCentral = fixedGridRhoFastjetAll.clone(
+    maxRapidity = cms.double(2.5)
+    )
```

Comment:
I suppose if that recipe includes
 ```from RecoJets/Configuration/RecoPFJets_cff  import fixedGridRhoFastjetCentral```
it would still kind of work.
Still, considering the inclusive  ``` keep *_fixedGridRho*_*_* ``` that we have in in the RecoJets_EventContent_cff, we will have to at least make a new Prompt version or wait until all PromptReco with pp is done to include this in the mainstream release.
The former is probably more feasible, but still not that desired.

Please put the definition of fixedGridRhoFastjetCentral in a separate file (in 74X only)